app/models/concerns/workflowable.rb

```diff
@@ -0,0 +1,17 @@
+module Workflowable
```

Comment:
There's another way of doing this, that doesn't extend the models. `DocumentWorkflow.new(risk).is_incomplete?`